packages/sindarian-ui/src/components/ui/otp-input/otp-input.stories.tsx

```
@@ -0,0 +1,93 @@
+      <InputOTPGroup>
+        <InputOTPSlot index={3} />
+        <InputOTPSlot index={4} />
+        <InputOTPSlot index={5} />
+      </InputOTPGroup>
+    </InputOTP>
+  )
+}
+
+export const Pattern: StoryObj<typeof InputOTP> = {
+  render: (args) => (
+    <InputOTP maxLength={6} pattern={REGEXP_ONLY_DIGITS} disabled={args.disabled}>
```

ESLint

[error] Replace `·maxLength={6}·pattern={REGEXP_ONLY_DIGITS}·disabled={args.disabled}` with `⏎······maxLength={6}⏎······pattern={REGEXP_ONLY_DIGITS}⏎······disabled={args.disabled}⏎····`

(prettier/prettier)